hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.tsx

```diff
@@ -63,14 +63,29 @@ export default function ActionRuleFormColumns({
             showErrors &&
             classification.classificationType === classificationTypeTBD
           }>
-          <option value={classificationTypeTBD}> Select... </option>
-          <option value="BankSourceClassification">Dataset Source</option>
-          <option value="BankIDClassification">Dataset ID</option>
-          <option value="BankedContentIDClassification">
+          <option key="tbd" value={classificationTypeTBD}>
```

Comment:
nit: can you make these 'classificationTypes' an enum-type thing like we did for `ActionerTypes` ?